platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetsPlugin.java

```diff
@@ -84,25 +85,38 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
 
         final Context applicationContext = cordova.getActivity().getApplicationContext();
 
-        pathToDatabase = applicationContext.getCacheDir().getAbsolutePath() + File.separator + PLUGIN_FOLDER;
-        pathToAssets = pathToDatabase + File.separator + ASSETS_FOLDER;
+        String pathToCache = applicationContext.getCacheDir().getAbsolutePath() + File.separator;
+
+        String pathToPlugin = pathToCache + PLUGIN_FOLDER;
+        pathToAssets = pathToPlugin + File.separator + ASSETS_FOLDER;
 
-        if (!createFolderIfRequired(pathToDatabase)) {
-            Log.e(TAG, "error -- unable to create folder: " + pathToDatabase);
+        if (!createFolderIfRequired(pathToPlugin)) {
+            Log.e(TAG, "error -- unable to create plugin folder: " + pathToPlugin);
             return;
         } else if (!createFolderIfRequired(pathToAssets)) {
-            Log.e(TAG, "error -- unable to create folder: " + pathToAssets);
+            Log.e(TAG, "error -- unable to create assets folder: " + pathToAssets);
             return;
         }
 
-        pathToDatabase += File.separator;
         pathToAssets += File.separator;
 
         setBlockSize();
 
-        wizAssetManager = new WizAssetManager(applicationContext, pathToDatabase);
+        removeDeprecatedDatabases(pathToCache);
+
+        initialized = true;
+    }
+
+    public void removeDeprecatedDatabases(String pathToCache) {
+        // Removing database version 6.x.x
+        String deprecatedDatabasePath = pathToCache + PLUGIN_FOLDER + File.separator + DEPRECATED_DATABASE_NAME;
+        File deprecatedDatabaseFile = new File(deprecatedDatabasePath);
+        deprecatedDatabaseFile.delete();
```

Comment:
Not a blocker but wouldn't be more elegant to test `deprecatedDatabaseFile.exists()` before and if that's the case then test if `deprecatedDatabaseFile.delete()` fails and send a `Log.e` in this case (instead of having two delete that are kind of failing everytime we start the app)?